web/docs/data-model/operations/queries.md

```diff
@@ -217,6 +221,47 @@ const doneTasks = await getFilteredTasks({ isDone: true })
 </TabItem>
 </Tabs>
 
+#### Using Queries on the server
+Calling a Query on the server is similar to calling it on the client.
+
+Here's what you have to do differently:
+ - Import Queries from `wasp/server/operations` instead of `wasp/client/operations`.
+ - Make sure you pass in a context object with the user to authenticated Queries.
+
+<Tabs groupId="js-ts">
+<TabItem value="js" label="JavaScript">
```

Comment:
These examples are "floating" without a function in which they are called. Is it an another action or query? Is it API? Also, how do I acquire an `AuthUser`?

Making the example more practical will be more ... practical for the users since they will want to do this in operations anyways and for that use case we have a good answer for them how to get the `AuthUser`.

Reply:
You got me.

I left it out on purpose because I didn't want to admit the API is still crippled.

Anyway, I'll improve what the comment says and make sure we tackle https://github.com/wasp-lang/wasp/issues/2066 as soon as possible.